include/openspace/interaction/joystickcamerastates.h

```diff
@@ -159,10 +193,35 @@ from_string(std::string_view string)
     if (string == "GlobalRoll Y") { return T::GlobalRollY; }
     if (string == "Pan X") { return T::PanX; }
     if (string == "Pan Y") { return T::PanY; }
+    if (string == "Property") { return T::Property; }
 
     throw RuntimeError("Unkonwn axis type '" + std::string(string) + "'");
 }
 
+template <>
+inline std::string to_string(
+    const openspace::interaction::JoystickCameraStates::JoystickType& value)
+{
+    using T = openspace::interaction::JoystickCameraStates::JoystickType;
+    switch (value) {
+    case T::JoystickLike: return "JoystickLike";
```

Comment:
Missing indentation;  the `case` labels are indented by one level relative to the `switch`